metricbeat/module/zookeeper/mntr/data.go

```diff
@@ -72,7 +72,7 @@ func eventMapping(response io.Reader, r mb.ReporterV2) {
 		if match := paramMatcher.FindStringSubmatch(scanner.Text()); len(match) == 3 {
 			fullEvent[match[1]] = match[2]
 		} else {
-			logp.Warn("Unexpected line in mntr output: %s", scanner.Text())
+			logger.Warn("Unexpected line in mntr output: %s", scanner.Text())
```

Comment:
In the past we made the decision that we try not to use `Warn` and always use either Error or Info to make it clear if user action is needed or not. I suggest we change this to Info.

I'm aware this is not introduced by this PR, but as we changed the line stumbled over it. We can change it in this PR or follow up.